packages/js/src/Modules/Verto/index.ts

```diff
@@ -20,7 +20,69 @@ export default class Verto extends BrowserSession {
     return isValidOptions(this.options);
   }
 
+  /**
+   * Makes a new outbound call.
+   *
+   * This method receives an object `options` with the following properties:
+   *
+   * @param options Options object for a new call.
+   * @param options.destinationNumber Extension to dial.
+   * @param options.callerNumber Number to use as the caller ID when dialling out to a phone number.
+   * @param options.callerName Name to use as the caller ID name when dialling out to a phone number.
+   * @param options.id The identifier of the Call.
+   * @param options.localStream If sets, the Call will use this stream instead of retrieving a new one.
```

Comment:
```suggestion
   * @param options.id The identifier of the call.
   * @param options.localStream If set, the call will use this stream instead of retrieving a new one.
```